src/coreclr/vm/codeman.cpp

```diff
@@ -3740,12 +3753,17 @@ BOOL InterpreterJitManager::LoadInterpreter()
     ICorJitCompiler* newInterpreter = NULL;
     m_interpreter = NULL;
 
+#ifdef FEATURE_STATICALLY_LINKED
```

Comment:
```suggestion
// If both JIT and interpret are available, statically link the JIT. Interpret can be loaded dynamically
// via config switch for testing purposes. 
#if defined(FEATURE_STATICALLY_LINKED) && !defined(FEATURE_JIT)
```

Reply:
We may also need a matching fix in makefiles to use the right static library when both JIT and interpreter are available, and static linking is on.